src/ApiPort/CommandLineOptions.cs

```diff
@@ -47,11 +48,11 @@ private static ICommandLineOptions ShowHelp(string suppliedCommand = null)
                 Console.WriteLine();
             }
 
-            Console.WriteLine("Available Commands:");
-
             foreach (var command in s_possibleCommands)
             {
-                Console.WriteLine($"- {command.Key}");
+                Console.WriteLine();
+                Console.WriteLine(new string('=', Math.Min(Console.WindowWidth, 100)));
```

Comment:
If you tried it out and it looks fine, that's good enough for me. Thanks for checking!